collectors/tc.plugin/README.md

```diff
@@ -172,7 +172,7 @@ And this is what you are going to get:
 
 ## QoS Configuration with tc
 
-First, setup the tc rules in rc.local using commands to assign different DSCP markings to different classids. You can see one such example in [github issue #4563](https://github.com/netdata/netdata/issues/4563#issuecomment-455711973). 
+First, setup the tc rules in rc.local using commands to assign different DHCP markings to different classids. You can see one such example in [github issue #4563](https://github.com/netdata/netdata/issues/4563#issuecomment-455711973). 
```

Comment:
It should be left as DSCP (stands for Differentiated Services Code Point).

Reply:
Perhaps we should change it to ‘QoS markings’? It’s obviously less precise, but both looks less like a typo and is less likely to be misinterpreted by someone who does not have a background in low-level networking.

Reply:
I think DSCP is well-known by people who know what QoS is. [COS](https://en.wikipedia.org/wiki/Class_of_service) is QoS markings too.

Reply:
Example, we are referring to, has `dsfield`. 

```
dsfield VAL_MASK_8
  IPv4 only. Match the packet header's DSCP/ECN
  field. Synonyms to this are tos and precedence.
```

Actually i agree - `QoS markings` will do.

This always was confusing to me - QoS, TOS, COS, IP Precedence, DSCP, ECN 🤪

Reply:
I'd be more than happy to change it to QoS, that seems like a reasonable choice - fwiw, there are ~2x as many hits for `"QoS markings"` than for `"dscp markings"` and they appear to be reasonable hits.

---

Alternatively, I'd suggest adding a couple of other instances of the word (even in this section). That's probably achievable by inlining some content from the link instead of forcing people to follow the link.

I tend to leave things alone if there are multiple hits and thus hint at being terms of art. Here, this is the _only_ instance of the token which made it suspect. (But not apparently enough for me to Google it.)

---

Reading the sentence, `Setup` should actually be `Set up`, so if this project has a documentation team, I'd suggest flagging this document to them for review :-).

---

Let me know which way you want to go. I wouldn't want to spend a lot of time trying to fix this if someone else w/ more domain knowledge could improve the section.

The easiest thing for me to do would be to drop the commit and file an issue asking for someone to improve the section (citing this thread).

Reply:
> if this project has a documentation team

Hey, that's me! :stuck_out_tongue: I don't have enough domain expertise on QoS to put in a vote either way, but I trust both @Ferroin and @ilyam8 to have the reader's best understanding in mind with their suggestions.

I'll make a note in my backlog to look at this doc, and this section, for additional improvements as a separate action/PR.

Reply:
I've pushed `QoS`. I think that'll enable people to move forward with this later.

I won't have any more time until next week.